.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
 - pip install aioredis
 - pip install cryptography
 - pip install pynacl
+- pip install pymongo
```

Comment:
Please remove pymongo and bson altogether, there is no point to have it here. You should be able to test custom serializer without introducing this dependency.